.svglintrc.js

```diff
@@ -473,6 +473,43 @@ module.exports = {
                 ignoreIcon(reporter.name, iconPath, $);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = "path-format";
+
+            const iconPath = $.find("path").attr("d");
+            if (!updateIgnoreFile && isIgnored(reporter.name, iconPath)) {
+              return;
+            }
+
+            const validPathFormatRegex = /^[Mm][MmZzLlHhVvCcSsQqTtAaEe0-9-,.\s]+$/;
+            if (iconPath.match(validPathFormatRegex) === null) {
+              let errorMsg = "Invalid path format: "
+
+              if ("Mm".indexOf(iconPath[0]) === -1) {
+                // doesn't start with moveto
+                errorMsg += `should start with \"moveto\" command (\"M\" or \"m\"), but starts with \"${iconPath[0]}\"`
+              } else {
```

Comment:
I don't think these two issues are exclusive, shoulnd't this be

```suggestion
              }
              if (/^[MmZzLlHhVvCcSsQqTtAaEe0-9-,.\s]+$/.test(iconPath)) {
```

At which point, the top-level if-block becomes redundant, I think :thinking:

Reply:
Note that in the `for` loop, I'm ignoring the first character, so it's exclusive.